### DISCUSSION TITLE
Meu projeto flutter não roda no emulador mas no chrome roda

### DISCUSSION THREAD

**Discussion by Ryan-Matheus** [Flutterando/forum#182] [Perguntas e Respostas] [answered]
Eu tentei diversas vezes e fiz muitos testes mas não consigo rodar o meu projeto de flutter no emulador do android, quando tento o emulador abre mas nao apresenta o projeto, sinceramente não sei muito mais o que fazer, alguem poderia dar uma ajuda? nem mesmo em device fisico eu consigo ver o app.
![imagem_2024-02-06_150709690](https://github.com/Flutterando/forum/assets/71937751/79822605-2b2a-43b4-8edb-a759a1247150)

**Comment by eduardoflorence** [comment]
Mostra o resultado do seu `flutter doctor`

**Comment by Ryan-Matheus** [comment]
![imagem_2024-02-06_155310424](https://github.com/Flutterando/forum/assets/71937751/74448814-dfbf-4ddc-a739-1768f6bd21ab)

**Comment by eduardoflorence** [comment]
Por favore, faz `flutter doctor -v` para ele mostrar os detalhes do 3 devices que estão conectados. Faz isso com o emulador aberto.

**Comment by Ryan-Matheus** [comment]
![imagem_2024-02-06_160425005](https://github.com/Flutterando/forum/assets/71937751/1be4530e-96ef-4e18-8a83-3eb1514b447e)

**Comment by eduardoflorence** [comment]
Você está escolhendo esse dispositivo "sdk gphone64 x86 64" para debugar? Está dando algum erro?

**Reply by Ryan-Matheus** [reply]
O dispositivo até abre e roda mas o aplicativo nao aparece dentro dele, fica apenas o simulador aberto comum

**Reply by eduardoflorence** [reply]
Deixa o emulador aberto e depois manda rodar de novo com ele já aberto

**Reply by Ryan-Matheus** [reply]
ja fiz isso mas tambem não funcionou

**Reply by eduardoflorence** [reply]
Nenhuma mensagem de erro?

**Reply by Ryan-Matheus** [reply]
nenhuma, o celular so roda normal no console tbm fica normal
![imagem_2024-02-06_165230652](https://github.com/Flutterando/forum/assets/71937751/1eba3a84-4bf5-429e-b912-ea1e0f8d0198)

**Reply by eduardoflorence** [reply]
na pasta do seu projeto, pelo terminal digita:

`flutter run`

Ele vai te mostrar os dispositivos conectados e te pedir para escolher o número daquele que deseja executar. Depois que escolher, veja se vai mostrar algum erro. Se não mostrar, digita o comando abaixo e repete o procedimento de escolher o dispositivo (esse com certeza mostra o erro):

`flutter run -v`

**Reply by Ryan-Matheus** [reply]
![imagem_2024-02-06_172343560](https://github.com/Flutterando/forum/assets/71937751/988b5aa3-e1a0-49a6-8d16-0d80956c6413)
recebi esse erro

**Reply by eduardoflorence** [reply, marked as answer]
O problema é essa pasta "Área de trabalho", que tem acento. Move a pasta do seu projeto para outra pasta e executa de lá

**Reply by Ryan-Matheus** [reply]
Muito obrigado resolveu aqui meu amigo, agradeço mesmo
